src/Admin/AbstractAdmin.php

```diff
@@ -72,6 +72,34 @@ abstract class AbstractAdmin extends AbstractTaggedAdmin implements AdminInterfa
             Doctrine\\\Orm|Doctrine\\\Phpcr|Doctrine\\\MongoDB|Doctrine\\\CouchDB
         )\\\(.*)@x';
 
+    private const ACTION_TREE = 1;
+    // private const ACTION_CREATE = 2; // not used now
```

Comment:
Why commenting this ?

Reply:
I suggest we reserve a bit for the Create action. But the conditions for the Create button are not described in method `AbstractAdmin::configureActionButtons()` and this constant is not used.

![image](https://user-images.githubusercontent.com/1954436/111956951-77216100-8afc-11eb-9e62-fb4e0d215244.png)

Would you rather reserve a bit, remove unused code, or leave dead code?

Reply:
If we need it, we could add it as the last bit so it's maybe not necessary to reserve a bit.
But if we reserve a bit, I prefer to create the const rather than having a comment.

Maybe @sonata-project/contributors think differently

Reply:
I removed the commented code.